index.html

```diff
@@ -60,45 +60,45 @@
 
     <section class="resume-section p-3 p-lg-5 d-flex d-column" id="about">
       <div class="my-auto">
-        <h1 class="mb-0">Lisa
-          <span class="text-primary">Simpson</span>
+        <h1 class="mb-0">Awet
+          <span class="text-primary">Fessha</span>
         </h1>
-        <div class="subheading mb-5">Glasgow · G1 (+44) 758-777-777 ·
-          <a href="mailto:name@email.com">name@email.com</a>
+        <div class="subheading mb-5">Glasgow · G1 (+44) 745-494-1123 ·
+          <a href="mailto:name@email.com">awetkifle94@gmail.com</a>
```

Comment:
Don't forget to change the `mailto` link too.

Reply:
Thanks for your feedback @tekul i will change it just now.